src/components/ElementBar/ColorPickerButton/ColorPicker.tsx

```diff
@@ -16,34 +16,37 @@
 
 import { Popover } from '@mui/material';
 import { unstable_useId as useId } from '@mui/utils';
-import { MouseEvent, useCallback, useState } from 'react';
-import { useTranslation } from 'react-i18next';
+import { ComponentType, MouseEvent, useCallback, useState } from 'react';
 import {
   useActiveElements,
   useElements,
   useWhiteboardSlideInstance,
 } from '../../../state';
-import { useLayoutState } from '../../Layout';
+import { ElementUpdate, Elements } from '../../../state/types';
 import { ToolbarSubMenu } from '../../common/Toolbar';
-import { ColorPickerIcon } from './ColorPickerIcon';
 import { ColorsGrid } from './ColorsGrid';
-import { calculateColorChangeUpdates } from './calculateColorChangeUpdates';
-import { extractFirstNonTransparentOrFirstColor } from './extractFirstNonTransparentOrFirstColor';
 
-export function ColorPicker() {
-  const { t } = useTranslation();
+export type ColorPickerProps = {
+  color?: string;
+  setColor?: (color: string) => void;
+  /** The icon component to be used for the colour picker in the element bar */
```

Comment:
Currently, most comments use `colour` and all code occurrences use `color`. This is obviously not an issue but would be good to have consistency here.

Being picky about color pickers is sort of a pun :)